### PR TITLE
Update special field decoration mode

### PR DESCRIPTION
**Pull request by david-allison**
## Purpose / Description

There are some cases of Note Editing, specifically if you have a 2 field card, and the second field is empty, where you should apply special decoration to the card based on the first field.  Updates and additions to these decorations are occasionally required due to the passage of time. Today is one of those times.

## Approach

Some effort was spent making sure that the Spanish language in particular was supported and that when considering a 26-letter alphabet, the desired decorations could readily be converted between our chosen storage mechanism and their regular representation with an easily reversible storage encoding by dividing the 26 letters into two 13-letter spaces with a rotating transform.

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/utils/NoteFieldDecorator.kt [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 package com.ichi2.utils
 
-import java.util.Random
+import java.util.*
```
Comment (ShridharGoel) [inline AnkiDroid/src/main/java/com/ichi2/utils/NoteFieldDecorator.kt:19]: Is this change required? Isn't it better to avoid wildcard imports?

**Comment by ShridharGoel** [comment]
Is there a documentation where I can read more about this to understand what this is about?

> Updates and additions to these decorations are occasionally required due to the passage of time.
Can you explain why?

File: AnkiDroid/src/main/java/com/ichi2/utils/NoteFieldDecorator.kt (hunk 1 above)
Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/utils/NoteFieldDecorator.kt:19]: I'm inclined to say if ktlint passes on it, then it's allowed. And if we feel differently then we should alter our lint settings.
I personally don't care too much about symbol imports these days, I've gotten lazy with IDE support that makes everything clickable and just generates imports for me automatically anyway :shrug:
